src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/PrisonDto.kt

```diff
@@ -32,4 +32,7 @@ data class PrisonDto(
 
   @Schema(description = "exclude dates", required = false)
   var excludeDates: Set<LocalDate> = mutableSetOf(),
+
+  @Schema(description = "prison user client", required = false)
+  val clients: List<PrisonUserClientDto> = mutableListOf(),
```

Comment:
should this be listOf() instead of mutableListOf?

Reply:
Yes really but because the type is List it wont have an add on it,... I will change